crates/uv-cli/src/lib.rs

```diff
@@ -2617,6 +2617,12 @@ pub struct RunArgs {
     #[arg(long, conflicts_with("only_group"))]
     pub group: Vec<GroupName>,
 
+    /// Exclude dependencies from the specified local dependency group.
+    ///
+    /// May be provided multiple times.
+    #[arg(long, conflicts_with("only_group"))]
```

Comment:
I don't think this conflicts? What if I do `--only-group foo --only-group bar --no-group foo`? Per previous discussions about overriding previous options, this seems nice to permit.

Reply:
Ok, I went back and forth on this.

Reply:
I trust your judgement, I was just surprised.

Reply:
It just added a bit of complexity to the implementation, so I had punted it. Makes sense though.